frontend/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+       // Fill in any extensions
```

Comment:
```suggestion
       "bradlc.vscode-tailwindcss"
```

Eigentlich nur das... @world-wide-weights/frontend Habt ihr noch andere Plugins in use welche sehr gut sind?

Reply:
added